packages/webapi/src/ponyfill.ts

```diff
@@ -7,6 +7,7 @@ import {
 import { Event, EventTarget } from 'event-target-shim'
 import { Blob, File } from 'fetch-blob/from.js'
 import { FormData } from 'formdata-polyfill/esm.min.js'
+import * as undici from 'undici'
```

Comment:
Trying to import the needed classes directly from `undici` failed with a weird error, probably because it's CJS. I couldn't  figure it out, but this works